lib/octokit/client/search.rb

```diff
@@ -64,6 +64,18 @@ def search_repositories(query, options = {})
       end
       alias :search_repos :search_repositories
 
+      # Search topics
+      #
+      # @param query [String] Search term and qualifiers
+      # @param options [Hash] Pagination options
+      # @option options [Integer] :page Page of paginated results
+      # @option options [Integer] :per_page Number of items per page
+      # @return [Sawyer::Resource] Search results object
+      # @see https://developer.github.com/v3/search/#search-topics
+      def search_topics(query, options = {})
+        search "search/topics", query, options
```

Comment:
It looks like we're missing the `application/vnd.github.mercy-preview+json` preview header for this request.

Reply:
totally missed it, thanks!